test/verify/check-realms

```diff
@@ -433,6 +433,11 @@ ExecStart=/bin/true
 """)
             self.machine.execute("ln -s /bin/true /usr/bin/chronyc")
 
+            # HACK: It now expects chrony.service and not chronyd.service
+            # https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1760235.html
```

Comment:
https://bugs.debian.org/968428 please

Reply:
hmm, that link does not work for me. Is it supposed to?

Reply:
Yes, it does work. I think one hour ago there was a short outage, though.

Reply:
ok, now it works. Seems I hit the outage.
Pushed here. Previous runs were all successful. Can we land this without any new tests since they retry ferociously ? I commented results in the next comment.